samples/msal-node-samples/auth-code/index.js

```diff
@@ -105,8 +120,21 @@ const getTokenAuthCode = function (scenarioConfig, clientApplication, port) {
     });
 
     app.get("/redirect", (req, res) => {
-        const tokenRequest = { ...requestConfig.tokenRequest, code: req.query.code };
-        const authCodeResponse = { nonce: req.session.nonce, code: req.query.code }
+        const state = req.query.state
+        const cachedState = req.session.state
+
+        /**
+         * MSAL Node provides the ClientApplication.validateSate API, which validates the state string received when 
+         * authorization code is returned.
+         *
+         */
+        clientApplication.validateState(state, cachedState);
```

Comment:
```suggestion
        clientApplication.validateState(req.query.state, req.session.state);
```
Can get rid of the const declarations above